.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,50 @@
+name: 🐛 Bug report
+
+# See the json schema to help you create / update this form
+# https://json.schemastore.org/github-issue-forms.json
+
+description: Create a report to help us improve
+
+labels: ["BUG"]
+
+body:
+
+  - type: markdown
+    attributes:
+      value: |
+        Please use the [MNE-Python forum](https://mne.discourse.group/) to ask (and answer)
+        concrete technical questions or about usage of MNE-BIDS.
+        Use this form if you have experienced bugs.
+
+  - type: textarea
+    attributes:
+      label: Description of the problem
+
+  - type: textarea
+    attributes:
+      label: Steps to reproduce
+      description: |
+        Replace this text with a code snippet or minimal working example
```

Comment:
```suggestion
        Paste here a code snippet or minimal working example
```

Reply:
Is there a way to make this field automatically formatted as python code? If so, maybe we should make a separate field for providing a URL for non-built-in datasets.